tests/integration/payment-profile.js

```diff
@@ -13,6 +13,21 @@ describe('Payment Profile', function(){
 		});
 	});
 
+	describe('Create with nonce', function(){
+		it('should create a payment profile with a nonce token', function(done){
+			
+			var options = {
+				paymentMethodNonce : '123',
+			}
+
+			tlPayment.service
```

Comment:
Good point. I was having trouble running the tests. ```make tests``` didn't appear to be working.